### PR TITLE
Deepthi add title with info icon to Teams page

### PR DESCRIPTION
**Pull request by deepthikannan**
# Description
image

…

## Main changes explained:
- Updated Teams page JSX: Added the Teams title and an EditableInfoModal with the info icon next to it.
- Updated Teams page CSS: Commented out the background: gray; rule in the .teams__overview--top class to remove the grey background affecting the Total Teams and Active Teams buttons for a cleaner layout.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to dashboard→ other links→ teams→…
6. verify The Teams title and the info icon are displayed correctly

## Screenshots or videos of changes:
image

### CODE REVIEW

**Comment by shivansh-nathani** [comment]
I can see the teams title added and have check for responsiveness of the page and it works as expected.
Screenshot 2024-10-07 at 9 39 20 PM

I had a quick question regarding the contents of the modal that pops up after clicking the i icon (Should it display the text as shown in image below or do we need to update that?)

Screenshot 2024-10-07 at 9 42 16 PM

**Comment by shivansh-nathani** [comment]
I have checked the PR and the title is present and I have also checked the website for responsiveness and it is working as expected.
Screenshot 2024-10-07 at 9 39 20 PM
I have a question regarding the modal contents when which opens up when i is clicked - should the content remain as shown in the image below or is it to be updated?
Screenshot 2024-10-07 at 9 42 16 PM

**Review by shivansh-nathani — CHANGES_REQUESTED** [review]
I have checked the changes and I am able to view the Teams title and I have also check for responsiveness and things look good.
Screenshot 2024-10-07 at 9 39 20 PM
I have a question on contents of the modal when i is clicked - do we also need to update it?
Screenshot 2024-10-07 at 9 42 16 PM

**Comment by Ankuriboh** [comment]
Works as described, UI and edit announcement functionalities also work.

**Review by Ankuriboh — APPROVED** [review]
Please ignore this approval, see below:

File: src/components/Teams/Teams.jsx [hunk 1]
```diff
@@ -116,10 +117,21 @@ class Teams extends React.PureComponent {
...
             <div className="container mt-3">
               {this.teampopupElements(allTeams)}
+              <div className="d-flex align-items-center">
+                <h3 style={{ display: 'inline-block', marginRight: 10 }}>Teams</h3>
+                <EditableInfoModal
+                  areaName="teamsInfoModal"
+                  areaTitle="Teams"
+                  fontSize={30}
+                  isPermissionPage={true}
+                  role={this.props.state.userProfile.role}
```
Comment (Ankuriboh) [inline src/components/Teams/Teams.jsx:127]: Should be `this.props.state?.auth?.user?.role`.

**Review by Ankuriboh — CHANGES_REQUESTED** [review]
UI and edit announcement functionalities work after fixing the problem in role props.

As also mentioned by @shivansh-nathani, there wasn't any button on the modal to indicate the message content could be edited or updated. I only knew that it was editable 
It would be better if we add an explicit button below the modal to show that it is actually editable. That's out of scope for this PR but fixing it could definitely a huge plus.

**Review by nikhilpittala16 — APPROVED** [review]
I have tested the PR and everything works as expected.

https://github.com/user-attachments/assets/f2e8fbf8-7bd6-4e00-a4e8-c325f78237a9

**Comment by tharunaa-1** [comment]
The message pops up as expected!

image

**Review by slidracoon72 — APPROVED** [review]
#2760 
@deepthikannan 
I have verified that the "Teams" title is displayed along with the "i" icon in both light and dark mode.

PR 2760 1
PR 2760 2

**Review by ishangoel17 — COMMENTED** [review]
Verified the PR and the changes are working as expected. Below is the screenshot for reference:
Capture2760

**Review by aaryaneil — DISMISSED** [review]
Tested the PR and the teams i icon works as expected for both light and dark mode.

https://github.com/user-attachments/assets/93fca159-df1b-4c0e-9f6b-29787845689c

**Review by CodewithKoushica — CHANGES_REQUESTED** [review]
The changes in this PR generally work well, and the new "Teams" title and info icon are displayed correctly. However, I noticed an issue on the smaller screen size. The buttons near this section become partially hidden which affects usability.

Suggestion: Please adjust the layout for smaller screen sizes (like iPhone 12) to ensure that all elements, including the buttons, remain visible and accessible. Other than that, everything looks good!

Screenshot 2024-10-24 at 5 18 55 PM

**Review by Muhideenthegreat — CHANGES_REQUESTED** [review]
Validation:

I confirm that the header was added to the page, with an info icon. 

Suggested Improvements:

- The search functionality can be standardized into a single, clear input field
- The page can be made more responsive, as it seems cramped on mobile devices
- The use of a clear color hierarchy for action buttons might be beneficial

Screenshot 2024-10-25 at 3 03 25 PM
Screenshot 2024-10-25 at 3 03 39 PM
Screenshot 2024-10-25 at 3 04 49 PM

**Review by SharadhaS24 — APPROVED** [review]
Tested the PR and verified that the team names are displayed correctly along with the team icon.

image
image

**Review by samarth9201 — APPROVED** [review]
LGTM!
Screenshot 2024-11-01 173451
Screenshot 2024-11-01 173458

**Review by ajitp4292 — APPROVED** [review]
Icon Works as expected
Screenshot 2024-11-02 at 7 59 52 PM

**Comment by deepthikannan** [comment]
Hi @Muhideenthegreat @shivansh-nathani @CodewithKoushica , the requested changes have been made.
The changes related to the information in the i icon, the option of editing the information is only accessible in the owner account which will be later reflected in the admin and other accounts. Kindly please verify by making any test changes. 

The changes related to the responsiveness of the page is fixed as separate PR https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2838 kindly please refer and verify

**Review by hqzhu0913 — APPROVED** [review]
The team icon works as expected.
Screenshot 2024-11-06 at 1 11 34 PM

**Review by CodewithKoushica — APPROVED** [review]
Page responsiveness remains unchanged, but I’m looking forward to seeing improvements in the new PR [#2838](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2838). Other than that, everything else is working well for me.
Screenshot 2024-11-06 at 5 51 56 PM

**Pull request merged**